A permission error caused `python` operator to fail when it was previously used
by another system user with the same set of requirements. There now is a one
Python environment per user and set of requirements.
